src/extension.ts

```diff
@@ -69,8 +69,8 @@ import {
   AuthProviderType,
 } from "./features/lightspeed/lightspeedUser";
 import {
-  PlaybookOutlineEvent,
-  PlaybookExplanationEvent,
+  PlaybookOutlineFeedbackEvent,
+  PlaybookExplanationFeedbackEvent,
```

Comment:
Because we are going to the thumbs up/down buttons from page 2 to page 3, I think we'd better call the event as `PlaybookGenerationFeedbackEvent`. wdyt?